django/db/models/fields/related.py

```diff
@@ -1258,6 +1258,16 @@ def _check_ignored_options(self, **kwargs):
                 )
             )
 
+        if self.remote_field.symmetrical and self._related_name:
+            warnings.append(
+                checks.Warning(
+                    'related_name has no effect on ManyToManyField '
+                    'with a symmetrical relationship (e.g. to "self").',
```

Comment:
Hey @partofthething — I agree about `e.g. to "self" but can I just suggest: 

```suggestion
                    'with a symmetrical relationship, e.g. to "self".',
```

— rather than the parentheses? 

(Happy to do this if you've run out of steam 😀)

Reply:
sounds good. on it.